### DISCUSSION TITLE
[DONE ✅]  Wallet setup finetuning

### DISCUSSION THREAD

**Discussion by dennisreimann** [btcpayserver/btcpayserver#2277] [General]
Adding this as a discussion to collect the TODOs that need to be addressed before we can ship the wallet setup redesign done in #2164. Feel free to add to this @pavlenex and @dstrukt, so that we can turn this into a proper issue I can work on.

Technical refactorings:
- [x] Remove old flow code (Stores.BTCLike Controller)
- [x] Code flow cleanups like @Kukks mentioned [here](https://github.com/btcpayserver/btcpayserver/pull/2164#discussion_r572023048)

Design and copy: 
- [x] Better icons for the create wallet options
- [ ] Improve information display on xpub import (table, [comment](https://chat.btcpayserver.org/btcpayserver/pl/6b4i4gthqfyw5x1ayd8dfcdoia))
- [ ] Potentially combine setup and modify page - see #2276
- [x] Improve the wording of the explanations
- [x] Incorporate [Nicolas' feedback on RPC import](https://github.com/btcpayserver/btcpayserver/pull/2164#issuecomment-777357793)

**Comment by pavlenex** [comment]
Thanks for making the list Dennis. I also added a [discussion on the Modify Page, which](https://github.com/btcpayserver/btcpayserver/discussions/2276) is better tackled separately imo, so just linking it.

**Comment by dstrukt** [comment]
Following up @dennisreimann we can probably close this one.
